tests/conditional_compilation/test_verify.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+""" Test to verify inference results.
+"""
+
+import numpy as np
+import glob
+
+
+def test_verify(test_id, model, artifacts, inaccuracy=0.000001):
+    out = artifacts / test_id
+    results = glob.glob(f"{out}*.npz")
+    assert len(results) == 2, "Too much or too few .npz files"
+    reference_results = np.load(results[0])
+    inference_results = np.load(results[1])
+    assert reference_results.keys() == inference_results.keys(), "Results have different layers"
```

Comment:
please, compare only sorted lists - `==` compares not only the content but the order of items too